### PR TITLE
Update migration script 

### PR DESCRIPTION
**Pull request by fuhuifang**
Update migration script to address:
- `pip install shrike` only installs logging
- `confidential_logging` is renamed to `compliant_logging`

### CODE REVIEW

File: shrike/__init__.py [hunk 1]
```diff
@@ -5,4 +5,4 @@
 Python utilities to aid "compliant experimentation" - training
 machine learning models without seeing the training data.
 """
-__version__ = "1.0.0-rc7"
+__version__ = "1.0.0-rc8"
```
Comment (haozhestat) [inline shrike/__init__.py:8]: We may not need to bump the version number, as the source code of shrike package itself has not changed in this PR.

**Review by haozhestat — CHANGES_REQUESTED** [review]

**Comment by haozhestat** [comment]
In the PR #21, we also make the following updates:
- enable_confidential_logging --> enable_compliant_logging
-  ConfidentialLogger --> CompliantLogger.

Could you address them in the `migration.py` as well?

**Review by haozhestat — APPROVED** [review]

**Comment by haozhestat** [comment]
@fuhuifang: By the way, could you please add a documentation on how to use the migration script? It could be in the README file or a separate doc page.

After this PR is merged, could you try the migration script in some customer repos, such as search relevance repo or smart compose repo? Thanks a lot.

**Comment by fuhuifang** [comment]
> @fuhuifang: By the way, could you please add a documentation on how to use the migration script? It could be in the README file or a separate doc page.
> 
> After this PR is merged, could you try the migration script in some customer repos, such as search relevance repo or smart compose repo? Thanks a lot.

Already tested against SearchRelevance repo locally. Will create a follow-up PR for instructions.

**Pull request merged**